spec/requests/api/service_requests_spec.rb

```diff
@@ -79,4 +79,120 @@ def expect_result_to_have_user_email(email)
+      api_basic_authorize collection_action_identifier(:service_requests, :read, :get)
+
+      run_get service_requests_url
+
+      expect(response).to have_http_status(:ok)
+      expect(response_hash).to include("name" => "service_requests", "count" => 1, "subcount" => 1)
+    end
+
+    it "a user can show their own request" do
+      service_request = FactoryGirl.create(:service_template_provision_request,
+                                           :requester   => @user,
+                                           :source_id   => template.id,
+                                           :source_type => template.class.name)
+      api_basic_authorize collection_action_identifier(:service_requests, :read, :get)
+
+      run_get service_requests_url(service_request.id)
+
+      expect(response).to have_http_status(:ok)
+      expect(response_hash).to include("id"   => service_request.id,
+                                       "href" => a_string_matching(service_requests_url(service_request.id)))
+    end
+
+    it "lists all the service requests if you are admin" do
+      allow_any_instance_of(User).to receive(:admin?).and_return(true)
+      other_user = FactoryGirl.create(:user)
+      service_request_1 = FactoryGirl.create(:service_template_provision_request,
+                                             :requester   => other_user,
+                                             :source_id   => template.id,
+                                             :source_type => template.class.name)
+      service_request_2 = FactoryGirl.create(:service_template_provision_request,
+                                             :requester   => @user,
+                                             :source_id   => template.id,
+                                             :source_type => template.class.name)
+      api_basic_authorize collection_action_identifier(:service_requests, :read, :get)
+
+      run_get service_requests_url
+
+      expected = {
+        "count"     => 2,
+        "subcount"  => 2,
+        "resources" => a_collection_containing_exactly(
+          {"href" => a_string_matching(service_requests_url(service_request_1.id))},
+          {"href" => a_string_matching(service_requests_url(service_request_2.id))},
+        )
+      }
+      expect(response).to have_http_status(:ok)
+      expect(response_hash).to include(expected)
+    end
+
+    it "an admin can see another user's request" do
+      allow_any_instance_of(User).to receive(:admin?).and_return(true)
+      other_user = FactoryGirl.create(:user)
+      service_request = FactoryGirl.create(:service_template_provision_request,
+                                           :requester   => other_user,
+                                           :source_id   => template.id,
+                                           :source_type => template.class.name)
+      api_basic_authorize collection_action_identifier(:service_requests, :read, :get)
```

Comment:
Yeah, as above it's the `@user` that comes from the init method. These helper methods all assume that's what we're dealing with :disappointed: